src/UI/Components/Diagram/helpers.ts

```diff
@@ -388,8 +395,15 @@ export const bundleInstances = (
   const embeddedInstances = deepCopiedMapToArray.filter(
     (instance) => !topServicesNames.includes(instance.service_entity),
   );
+  return topInstances.map((instance) => {
```

Comment:
topInstances, are those the root instances?

Reply:
topInstance are non-embedded entities, as they at the top and receive eventual embedded ones